<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1168</width>
    <height>876</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Qt+VTK Sekeleton</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <property name="dockNestingEnabled">
   <bool>true</bool>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>245</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>967</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Visualization</string>
       </attribute>
       <attribute name="toolTip">
        <string>Color presets, tuning, scene elements...</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>221</width>
          <height>91</height>
         </rect>
        </property>
        <property name="title">
         <string>Point of view</string>
        </property>
        <widget class="QRadioButton" name="radioButtonXY">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>100</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>XY plane</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QRadioButton" name="radioButtonXZ">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>100</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>XZ plane</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="radioButtonYZ">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>60</y>
           <width>100</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>YZ plane</string>
         </property>
        </widget>
       </widget>
       <widget class="QCheckBox" name="checkVisualizeAxis">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>100</y>
          <width>201</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Visualize axis</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkVisibility">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>130</y>
          <width>201</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Show scene</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QVTKWidget" name="vtkContext" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1168</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Fi&amp;le</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuElements">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>&amp;Visualization</string>
    </property>
    <widget class="QMenu" name="menuFixed_Point_of_View">
     <property name="title">
      <string>&amp;Fixed Point of View</string>
     </property>
     <addaction name="actionX_Y"/>
     <addaction name="actionX_Z"/>
     <addaction name="actionY_Z"/>
    </widget>
    <addaction name="actionVisibility"/>
    <addaction name="actionVisualize_Axis"/>
    <addaction name="menuFixed_Point_of_View"/>
    <addaction name="actionTake_snapshot"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_Skeleton"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuElements"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="actionShort_Axis">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Short Axis</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+K</string>
   </property>
  </action>
  <action name="actionVisualize_Axis">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Visualize Axis</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionTractography">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Tractography</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+J</string>
   </property>
  </action>
  <action name="actionSeedRandom_Mode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Random</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionInteractive">
   <property name="text">
    <string>Toggle interaction mode</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionSeedAutomatic_Mode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Automatic</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionAbout_Skeleton">
   <property name="text">
    <string>&amp;About Qt+VTK Skeleton</string>
   </property>
  </action>
  <action name="actionSave_StreamLines_for_Matlab">
   <property name="text">
    <string>&amp;Save StreamLines for MATLAB</string>
   </property>
  </action>
  <action name="actionClipping_box">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Clipping box</string>
   </property>
  </action>
  <action name="actionReset_box">
   <property name="text">
    <string>&amp;Reset box</string>
   </property>
  </action>
  <action name="actionX_Y">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;X-Y</string>
   </property>
  </action>
  <action name="actionX_Z">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>X-&amp;Z</string>
   </property>
  </action>
  <action name="actionY_Z">
   <property name="text">
    <string>&amp;Y-Z</string>
   </property>
  </action>
  <action name="actionSeeding_Box">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Seeding Box</string>
   </property>
  </action>
  <action name="actionSave_StreamLines_for_VTK">
   <property name="text">
    <string>Save StreamLines for VTK</string>
   </property>
  </action>
  <action name="actionTake_snapshot">
   <property name="text">
    <string>Take snapshot</string>
   </property>
  </action>
  <action name="actionVisibility">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show scene</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>qvtkwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actionOpen</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>FileOpen()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>154</x>
     <y>322</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>FileExit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>154</x>
     <y>322</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionX_Y</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>SetViewXY()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>430</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionX_Z</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>SetViewXZ()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>430</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionY_Z</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>SetViewYZ()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>430</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionTake_snapshot</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>Snapshot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>583</x>
     <y>437</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionVisualize_Axis</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>ToggleVisualizeAxis(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>583</x>
     <y>437</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonXY</sender>
   <signal>released()</signal>
   <receiver>MainWindow</receiver>
   <slot>SetViewXY()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>94</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>583</x>
     <y>437</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonXZ</sender>
   <signal>released()</signal>
   <receiver>MainWindow</receiver>
   <slot>SetViewXZ()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>94</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>583</x>
     <y>437</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonYZ</sender>
   <signal>released()</signal>
   <receiver>MainWindow</receiver>
   <slot>SetViewYZ()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>94</x>
     <y>123</y>
    </hint>
    <hint type="destinationlabel">
     <x>583</x>
     <y>437</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkVisualizeAxis</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>ToggleVisualizeAxis(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>125</x>
     <y>163</y>
    </hint>
    <hint type="destinationlabel">
     <x>583</x>
     <y>437</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkVisibility</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>ToggleVisibility(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>125</x>
     <y>193</y>
    </hint>
    <hint type="destinationlabel">
     <x>583</x>
     <y>437</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionVisibility</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>ToggleVisibility(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>583</x>
     <y>437</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>FileOpen()</slot>
  <slot>FileExit()</slot>
  <slot>ToggleVisualizeAxis(bool)</slot>
  <slot>ShowAboutDialog()</slot>
  <slot>SetViewXY()</slot>
  <slot>SetViewXZ()</slot>
  <slot>SetViewYZ()</slot>
  <slot>Snapshot()</slot>
  <slot>ToggleVisibility(bool)</slot>
 </slots>
</ui>
